### DISCUSSION TITLE
Delete Button

### DISCUSSION THREAD

**Discussion by de-johannes** [DirectoryLister/DirectoryLister#641] [Ideas & Features]
Hey,

I love DirectoryLister, as it is the only product to me, as no-coder, who fulfills nearly all the needs in cooperation with jitsi. 

Jitsi Jibri records a session, saves to file, changes the file name and moves the files into a new folder (corresponding to the name of the file). On a Website one can enter some infos and a script generates an url to the right directory in DirectoryLister, where the recorded session waits for its download.

The only thing I need, is the option to delete a file - and i am not able to edit the code myself to get a new button next to the FileInfo. So a user can download the file and delete it immediately.

Is it possible you add the feature or help me out with the code? 

Regards,
Johannes

**Comment by PHLAK** [comment]
The ability to delete files from the Directory Lister UI is _not_ a feature we will be adding, sorry.
